index.html

```diff
@@ -135,7 +135,7 @@ <h3 class="h-form">.</h3>
       a feature that you need built or a project that needs coding.
        I’d love to help with it 
     </P>
-    <form action="https://formspree.io/f/mbjwogbd" method="post">
+    <form class="form" action="#form-section" method="post" id="signup">
```

Comment:
- [ ] [OPTIONAL] Please, fix the `action` for your contact form, that it could send messages to the input mail address. Actually, the action is set as `#form-section`
![2022-03-04 22 24 08](https://user-images.githubusercontent.com/49064106/156820409-fd8ed042-defc-407b-ad9a-9594b3f7ad8e.gif)